Src/ILGPU/ILGPU.csproj

```diff
@@ -7,6 +7,11 @@
         <Configurations>Debug;Release</Configurations>
     </PropertyGroup>
 
+    <PropertyGroup Condition="'$(Experimental)' == 'true'">
+        <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net47;netcoreapp2.0;netstandard2.1;net5.0</TargetFrameworks>
```

Comment:
Question: `netcoreapp2.0`?

Reply:
Rather than trying to remove it using this PR, I've left `netcoreapp2.0` alone.
Can raise a separate PR to that it's a bit more obvious in our commit history.

Reply:
Makes sense 🤓